test/src/loaders/workspace_uploader/workspace_uploader_test.py

```diff
@@ -341,6 +441,67 @@ def test_upload_assembly_to_workspace(setup_and_teardown):
     )
 
 
+def test_upload_genome_to_workspace(setup_and_teardown):
+    params = setup_and_teardown
+    genome_name = GEMOME_NAMES[0]
+    assembly_name = ASSEMBLY_NAMES[0]
+    genome_coll_src_dir = Path(params.collection_source_dir) / genome_name
+    genome_coll_src_dir.mkdir(parents=True)
+    container_dir = Path(params.tmp_dir) / "kb/module/work/tmp"
+    container_dir.mkdir(parents=True)
+    shutil.copy(params.target_files[0], container_dir)
+
+    genome_obj_info = GENOME_OBJ_INFOS[0]
+    assembly_obj_info = ASSEMBLY_OBJ_INFOS[0]
+
+    gfu = create_autospec(GenomeFileUtil, spec_set=True, instance=True)
+    gfu.genbanks_to_genomes.return_value = {"results": [{"genome_ref": "72231/61/1",
+                                                         "assembly_ref": "72231/60/1",
+                                                         "assembly_path": container_dir / assembly_name,
+                                                         "assembly_info": assembly_obj_info,
+                                                         "genome_info": genome_obj_info}]}
+    genome_tuple = workspace_uploader.WSObjTuple(
+        genome_name, genome_coll_src_dir, container_dir
+    )
```

Comment:
Isn't container_dir supposed to be a path?

Reply:
👍

Reply:
:+1: